package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mapbox/spritezero",
-  "version": "7.0.0",
+  "version": "7.0.1",
```

Comment:
Nit: Since functionality is being added, maybe consider a minor update instead of a patch update?

Reply:
Good call. Since I've already released however I'll hold off from changing this.